WebRoot/css/zm.css

```diff
@@ -7279,4 +7279,32 @@ input[type="radio"] + label {
 
 .ZmWhiteBlackViewNumberUsed {
 	@WhiteBlackListViewItemsNumberUsed@
-}
\ No newline at end of file
+}
+
+.ZOptionsSectionMain .ZmPrefMailBlockMessage {
+	@PrefMailBlockMessageCell@
+}
+
+.ZOptionsLabelTop.ZmPrefMailComposeLabel {
+	@PrefMailComposeLabel@
+}
+
+.ZmPrefMailComposeContent .ZOptionsLabelNarrow {
+	@PrefMailComposeRowLabel@
+}
+
+.ZmPrefMailComposeContent .ZOptionsField {
+	@PrefMailComposeRowValueWrapper@
+}
+
+.ZmPrefMailComposeContent .ZOptionsField .ZSelect > table {
+	@PrefMailComposeRowSelectWidget@
+}
+
+.ZmPrefMailComposeContent .ZWidgetTable {
```

Comment:
Changed the classname to ZmPrefMailComposeColorPicker .
And, selector is now .ZmPrefMailComposeContent .ZmPrefMailComposeColorPicker table 
This intend to target color picker